SCClassLibrary/Common/GUI/PlusGUI/Control/server-scope.sc

```diff
@@ -45,7 +45,7 @@
 		bytes = synthDef.asBytes;
 		synthMsg = synth.newMsg(server, [\i_out, outbus, \out, outbus], \addToHead);
 		server.sendMsg("/d_recv", bytes, synthMsg);
-		server.scope(numChannels, outbus, bufsize, zoom, outUGen.rate);
+		server.scope(numChannels, outbus, bufsize, zoom, outUGen.rate).bounds_(bounds); 
```

Comment:
I have found a problem when using this PR's change: #6674
This issue occurs due to the following part:
```
.bounds_(bounds);
```
in line 48. I have no idea how to fix it.